test cases/frameworks/17 vulkan/vulkanprog.c

```diff
@@ -0,0 +1,35 @@
+#include <vulkan/vulkan.h>
+#include <stdio.h>
+
+int main()
+{
+    VkApplicationInfo application_info = {
+        VK_STRUCTURE_TYPE_APPLICATION_INFO,
+        	NULL,
+        	"sway",
+        	VK_MAKE_VERSION(1,0,0),
+        	"wlroots",
+        	VK_MAKE_VERSION(1,0,0),
+        	VK_MAKE_VERSION(1,0,0)
+    };
+
+    VkInstanceCreateInfo instance_create_info = {
+        	VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO,
+        	NULL,
+        	0,
+        	&application_info,
```

Comment:
The field `pApplicationInfo` is optional for `VkInstanceCreateInfo`. You can set it to `NULL` and save some code.